pages.ts

```diff
@@ -63,12 +82,12 @@ export async function loadLivePage(
 export const fetchPageFromPathname = async (
   path: string,
   siteId: number,
-): Promise<PageWithParams | null> => {
+): Promise<PageWithParams[] | null> => {
   const { data: pages, error } = await getSupabaseClient()
     .from<Page & { site: number }>("pages")
     .select("id, name, data, path, state")
     .eq("site", siteId)
-    .eq("state", "published");
+    .in("state", ["published", "draft"]);
```

Comment:
Why? Shouldn't we access `drafts` only when we have `?pageId=x`?

Reply:
for now we're contemplating not having a separate `experiment` state, remember? so we don't have to have two sources of truth - being in the flag is the truth.